doc/source/notes/5.0.0.rst

```diff
@@ -202,6 +202,8 @@ Deprecations
 * Swift (currently available as Technical Preview) will be removed in favor of Ceph RGW
 * Trove (currently available as Technical Preview) will be removed in favor of Kubernetes
   database operators
+* Skydive will be removed in the future, the project is not maintained anymore, last commit
```

Comment:
Skydive is also marked as preview, maybe use the same comment as for the previous deprecations? https://github.com/osism/documentation/blob/450b9eb5dc8db556e273a2656c9a601723a4c783/source/configuration/environments/openstack/skydive.rst#L7